manifests/0000_70_cluster-network-operator_01_crd.yaml

```diff
@@ -299,6 +299,17 @@ spec:
                 nullable: true
                 type: object
                 x-kubernetes-preserve-unknown-fields: true
+              useMultiNetworkPolicy:
```

Comment:
You need to update this in openshift/api and then revendor.

Reply:
As far as I checked. openshift/api is updated at https://github.com/openshift/cluster-network-operator/commit/0edcc38777cac2bb9abc6ce78906e80b2e78c9a1 and now we have `useMultiNetworkPolicy` in https://github.com/openshift/cluster-network-operator/blob/master/vendor/github.com/openshift/api/operator/v1/types_network.go#L63-L71

Reply:
Hence we don't need additional changes for that.